staging/src/k8s.io/apiserver/pkg/util/flowcontrol/reqmgmt-config.go

```diff
@@ -0,0 +1,246 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import (
+	"fmt"
+	"math"
+	"sort"
+	"sync"
+	"time"
+
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/wait"
+	fq "k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing"
+	kubeinformers "k8s.io/client-go/informers"
+	cache "k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog"
+
+	rmtypesv1a1 "k8s.io/api/flowcontrol/v1alpha1"
+)
+
+// initializeConfigController sets up the controller that processes
+// config API objects.
+func (reqMgmt *requestManagementSystem) initializeConfigController(informerFactory kubeinformers.SharedInformerFactory) {
+	reqMgmt.configQueue = workqueue.NewNamedRateLimitingQueue(workqueue.NewItemExponentialFailureRateLimiter(200*time.Millisecond, 8*time.Hour), "req_mgmt_config_queue")
+	fci := informerFactory.Flowcontrol().V1alpha1()
+	pli := fci.PriorityLevelConfigurations()
+	fsi := fci.FlowSchemas()
+	reqMgmt.plLister = pli.Lister()
+	reqMgmt.plInformerSynced = pli.Informer().HasSynced
+	reqMgmt.fsLister = fsi.Lister()
+	reqMgmt.fsInformerSynced = fsi.Informer().HasSynced
+	pli.Informer().AddEventHandler(reqMgmt)
+	fsi.Informer().AddEventHandler(reqMgmt)
+}
+
+func (reqMgmt *requestManagementSystem) triggerReload() {
+	klog.V(7).Info("request-management system reloading")
```

Comment:
To be precise, this is one step away from actually doing the reload.

Reply:
updated to `triggered request-management system reloading`